code/game/machinery/casino_ch.dm

```diff
@@ -0,0 +1,900 @@
+		"tonic", "beer", "kahlua", "whiskey", "wine", "vodka", "gin", "rum", "tequilla", "vermouth", "cognac",
+		"ale", "mead", "bitters", "champagne", "singulo", "doctorsdelight", "nothing", "banana", "honey", "egg",
+		"coco", "cherryjelly", "carrot", "apple", "tomato", "nutbutter", "soymilk", "grenadine", "gingerale", "royrogers",
+		"patron", "goldschlager", "gelatin", "melonliquor", "bluecuracao", "thirteenloko", "deadrum", "sake", "acidspit",
+		"amasec", "beepsky_smash", "atomicbomb", "nuka_cola", "threemileisland", "manhattan_proj", "psilocybin", "moonshine",
+		"specialwhiskey", "unathiliquor", "winebrandy", "snaps"
+		) // Re-add matcha latte once its in code, probably add more stuff to list as well
+
+/obj/machinery/chemical_dispenser/deluxe/full
+	spawn_cartridges = list(
+			/obj/item/weapon/reagent_containers/chem_disp_cartridge/water,
+			/obj/item/weapon/reagent_containers/chem_disp_cartridge/ice,
+			/obj/item/weapon/reagent_containers/chem_disp_cartridge/coffee,
+			/obj/item/weapon/reagent_containers/chem_disp_cartridge/cream,
+			/obj/item/weapon/reagent_containers/chem_disp_cartridge/tea,
+			/obj/item/weapon/reagent_containers/chem_disp_cartridge/icetea,
+			/obj/item/weapon/reagent_containers/chem_disp_cartridge/cola,
+			/obj/item/weapon/reagent_containers/chem_disp_cartridge/smw,
+			/obj/item/weapon/reagent_containers/chem_disp_cartridge/dr_gibb,
+			/obj/item/weapon/reagent_containers/chem_disp_cartridge/spaceup,
+			/obj/item/weapon/reagent_containers/chem_disp_cartridge/tonic,
+			/obj/item/weapon/reagent_containers/chem_disp_cartridge/sodawater,
+			/obj/item/weapon/reagent_containers/chem_disp_cartridge/lemon_lime,
+			/obj/item/weapon/reagent_containers/chem_disp_cartridge/sugar,
+			/obj/item/weapon/reagent_containers/chem_disp_cartridge/orange,
+			/obj/item/weapon/reagent_containers/chem_disp_cartridge/lime,
+			/obj/item/weapon/reagent_containers/chem_disp_cartridge/watermelon,
+			/obj/item/weapon/reagent_containers/chem_disp_cartridge/lemon,
+			/obj/item/weapon/reagent_containers/chem_disp_cartridge/coffee,
+			/obj/item/weapon/reagent_containers/chem_disp_cartridge/cafe_latte,
+			/obj/item/weapon/reagent_containers/chem_disp_cartridge/soy_latte,
+			/obj/item/weapon/reagent_containers/chem_disp_cartridge/hot_coco,
+			/obj/item/weapon/reagent_containers/chem_disp_cartridge/milk,
+			/obj/item/weapon/reagent_containers/chem_disp_cartridge/cream,
+			/obj/item/weapon/reagent_containers/chem_disp_cartridge/tea,
+			/obj/item/weapon/reagent_containers/chem_disp_cartridge/ice,
+			/obj/item/weapon/reagent_containers/chem_disp_cartridge/mint,
+			/obj/item/weapon/reagent_containers/chem_disp_cartridge/orange,
+			/obj/item/weapon/reagent_containers/chem_disp_cartridge/lemon,
+			/obj/item/weapon/reagent_containers/chem_disp_cartridge/lime,
+			/obj/item/weapon/reagent_containers/chem_disp_cartridge/berry,
+			/*/obj/item/weapon/reagent_containers/chem_disp_cartridge/matcha_latte,*/ //Not ported yet either - Jack
```

Comment:
Travis is screaming about this line not having a comma at the very end of it as is normal for a list. A simple addition of said missing comma at the end of this line will fix Travis's screeching.